doc/source/data/batch_inference.rst

```diff
@@ -173,7 +174,7 @@ For more in-depth examples for your use case, see :ref:`batch_inference_examples
             predictions.show(limit=1)
 
         .. testoutput::
-            :options: +MOCK
+            :skipif: True
```

Comment:
did the MOCK option not work? or what was the issue with these tests? cc @bveeramani

Reply:
the :book: Documentation CI run complains about not knowing it. if `skipif` works as intended as well, we can use it.

fwiw, I didn't find any Sphinx docs  for this MOCK option either.